### PR TITLE
Fix the animations for the platformer sample

### LINKED ISSUE #22: Platformer player animations not working
Hi,
I had a problem while trying to tests MonoGame Extended according to example codes and animation sprite didn't work for me, so I've decided to try the sample code (I've assumed earlier that it works) and animation for player is not working for some reason.
At  the same time animations for blue guy works fine.

### PR DESCRIPTION
**Pull request by kaltinril**
This fixes #22

### CODE REVIEW

**Comment by kaltinril** [comment]
https://github.com/user-attachments/assets/d8442c65-cba9-4ac2-948e-2eb7afddbed7

File: src/Games/Platformer/Systems/PlayerSystem.cs [hunk 1]
```diff
@@ -86,44 +86,51 @@ public override void Process(GameTime gameTime, int entityId)
...
-                    sprite.SetAnimation("jump");
+                    if (sprite.CurrentAnimation != "jump")
+                        sprite.SetAnimation("jump");
                     //sprite.Play("jump");
                     break;
                 case State.Walking:
-                    sprite.SetAnimation("walk");
+                    if (sprite.CurrentAnimation != "walk")
+                        sprite.SetAnimation("walk");
                     //sprite.Play("walk");
```
Comment (AristurtleDev) [inline src/Games/Platformer/Systems/PlayerSystem.cs:96]: Can we remove these commented out lines if they are no longer needed?

**Review by AristurtleDev — CHANGES_REQUESTED** [review]
If you wouldn't mind cleaning up those commented out lines in the switch block, that would be it.

Thanks again @kaltinril

**Review by AristurtleDev — APPROVED** [review]

**Comment by kaltinril** [comment]
@AristurtleDev  sounds good, I've removed the commented out items from someone's previous refactor.  I didn't want to touch them because I wasn't sure why they were left in.  There are more in the GameMain.cs that I wasn't sure about.  I know that _world is being updated by the Components (ECS) automatically, so those could probably be removed also, I just didn't want to touch them because I didn't know about it much

**Pull request merged**